doc/media/openage/terrain_format_spec.md

```diff
@@ -0,0 +1,340 @@
+docs.
+
+```
+imagefile 0 "/{aoe2_base@openage}/sand.png"  # absolute path with modpack identifier
+imagefile 1 "/{aoe2_base}/sand.png"          # absolute path with modpack alias
+```
+
+Absolute paths are the only way to reference image resources from other
+modpacks.
+
+
+#### Example
+
+```
+imagefile 0 "grass.png"
+imagefile 1 "../../sand.png"
+imagefile 2 "/{aoe2_base}/graphics/grass.png"
+```
+
+
+### `blendtable`
+
+Defines a blending table that is used for looking up a blending mask. The
+blending mechanism is described in more detail in the [blendomatic](/doc/media/blendomatic.md)
+documentation. Blending masks/tables are **optional** and do not need to be used.
+
+Parameter  | Type   | Optional | Default value
+-----------|--------|----------|--------------
+table_id   | int    | No       | -
+filename   | string | No       | -
+priority   | int    | No       | -
+blend_mode | int    | No       | -
+
+**table_id**<br>
+Reference ID for the blending table used in this file. IDs should start at `0`.
+
+**filename**<br>
+Path to the blending table definition on the filesystem.
+
+There are two ways to specify a path: relative and absolute. Relative
+paths are relative to the location of the terrain definition file. They
+can only refer to image resources that are in the same modpack.
+
+```
+blendmask 0 "blend0.bltable" 10 3        # blend0.bltable is in the same folder as the terrain file
+blendmask 1 "./blend0.bltable" 10 3      # same as above, but more explicit
+blendmask 2 "media/blend1.bltable" 10 3  # blend1.bltable is in the subfolder media/
+```
+
+Absolute paths start from the (virtual) modpack root (the path where all
+modpacks are installed). They always begin with `/` followed by either
+a modpack identifier or a shortened modpack alias enclosed by `{}`. For
+information on modpack identifiers and aliases see the [TODO]() docs.
+
+```
+blendmask 0 "/{aoe2_base@openage}/blend0.bltable" 10 3  # absolute path with modpack identifier
+blendmask 1 "/{aoe2_base}/blend0.bltable" 10 3         # absolute path with modpack alias
+```
+
+Absolute paths are the only way to reference image resources from other
+modpacks.
+
+**priority**<br>
+Decides which blending table of the two adjacent terrain textures is selected.
+The table referenced by the terrain with the highest priority value will be picked.
+If two adjacent terrains have equal priority, the blending table of the terrain
+with a lower x coordinate value is selected. If the x coordinate value is also
+equal, the blending table of the terrain with the lowest y coordinate is selected.
+
+```
+# grass.terrain
+...
+blendmask 0 "blend0.bltable" 12 2
+...
+
+# sand.terrain
+...
+blendmask 0 "blend1.bltable" 10 3
+...
+
+-> grass.terrain's blending table has a higher blending priority.
+   Its blending table definition is therefore used when grass.terrain
+   and sand.terrain are adjacent to each other.
```

Comment:
for easier understandability i'd use the same mode here, and blend e.g. grass with forest, then only the priority should differ. the priority is also used (afair) to select which terrain tile flows on the other, i.e. which has flowing priority.

Reply:
done